backend/app/controllers/backend/albums_controller.rb

```diff
@@ -87,7 +87,7 @@ def set_album
       end
 
       def set_albums
-        @albums = Album.albums(@search, @index_items_limit * @page)
+        @albums = Album.albums(params, @index_items_limit * @page)
```

Comment:
Normally that goes in the controller. So the ApplicationController of the backend is probably the best way, as it's shared by other controllers.